examples/tv-casting-app/android/App/app/src/main/java/com/matter/casting/ConnectionExampleFragment.java

```diff
@@ -152,9 +228,124 @@ public void handle(MatterError err) {
             });
   }
 
+  private void displayPasscodeInputDialog(Context context) {
+    AlertDialog.Builder builder = new AlertDialog.Builder(context);
+
+    LayoutInflater inflater = LayoutInflater.from(context);
+    View dialogView = inflater.inflate(R.layout.custom_passcode_dialog, null);
+
+    // Set up the input dialog with the default passcode
+    final EditText input = dialogView.findViewById(R.id.passcode_input);
+    input.setText(DEFAULT_COMMISSIONER_GENERATED_PASSCODE_STRING);
```

Comment:
We don't need a separate DEFAULT\_COMMISSIONER\_GENERATED\_PASSCODE\_STRING since we already have DEFAULT\_COMMISSIONER\_GENERATED\_PASSCODE\_INT. Just use `"" + DEFAULT_COMMISSIONER_GENERATED_PASSCODE_INT` if needed.